src/search-schema/data/definitions/enums/assay_types.yaml

```diff
@@ -287,11 +293,11 @@ salmon_sn_rnaseq_10x:
     primary: false
     vitessce-hints: ['is_sc', 'rna']
 
-# Slide-seq:
-#     description: ????
-#     alt-names: []
-#     primary: true
-#     vitesse-hints: [[]
+Slide-seq:
+    description: Slide-seq
+    alt-names: []
+    primary: true
+    vitesse-hints: []
```

Comment:
I think we can add `rna` here since this is spatially resolved rna-seq

Reply:
fixed!